jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -30,6 +30,10 @@
 import java.util.Map;
 import java.util.function.Function;
 
+/*[IF JFR_SUPPORT]*/
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
```

Comment:
This only applies to JDK17+.

Reply:
JFR_SUPPORT tag for 11 and 17 were also added through following PR's

11: https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/811
17: https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/390